### PR TITLE
Added a schema generator

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add in the project the tools used before to generate a database schema.
| Type?         | new feature
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Follow the instructions in the readme, using MySQL Workbench I was able to generate a schema like the one described [here](https://github.com/PrestaShop/docs/pull/115#issuecomment-416944280). 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10197)

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
I create the pull request only because I think this script will be lost else :/

The only *right* way is to stop to remove the foreign keys during the installation: I believe this hack was introduced during ~~1.7.4.2~~ 1.7.1.1, I'd like to know why and if this is something we can fix.

WDYT?

**Comment by Quetzacoalt91** [comment]
PrestaShop 1.7.1.1, a test exists about this in the 1-click upgrade module:

```php
        if (version_compare($this->destinationPsVersion, '1.7.1.1', '>=')) {
            $schemaUpgrade = new \PrestaShopBundle\Service\Database\Upgrade();
            $outputCommand = 'prestashop:schema:update-without-foreign';
        } else {
            $schemaUpgrade = new \PrestaShopBundle\Service\Cache\Refresh();
            $outputCommand = 'doctrine:schema:update';
        }
```

**Comment by Quetzacoalt91** [comment]
I guess we should remove these files on releases. Can you please add the path to the ignored paths? https://github.com/PrestaShop/PrestaShop/blob/e9d7fd784f41332eb303dd6247969b3941ff0f6f/tools/build/Library/ReleaseCreator.php#L88-L131

**Comment by mickaelandrieu** [comment]
@Quetzacoalt91 done!

File: tools/foreignkeyGenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+removed during the Shop installation.
+
+This script have known bugs, must be updated manually everytime we update a table and is not tested.
+
+Once you have executed it, you can use modelisation tools like MySQL Workbench to generate an EER or MCD diagram.
+
+## How to use it?
+
+```
+./tools/foreignKeyGenerator/foreign.php
```
Comment (Quetzacoalt91) [inline tools/foreignkeyGenerator/README.md]: Case is important on some OS :)

```suggestion
./tools/foreignkeyGenerator/foreign.php
```

Comment (mickaelandrieu) [reply tools/foreignkeyGenerator/README.md]: Thanks :)

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @mickaelandrieu

**Pull request merged**